### PR TITLE
React Basics: Attributes

### PR DESCRIPTION
**Pull request by PReichetanz**
rendered: https://github.com/neuefische/web-exercises/tree/react-basics-attributes/sessions/react-basics/attributes

- [CSB](https://codesandbox.io/s/github/neuefische/web-exercises/tree/react-basics-attributes/sessions/react-basics/attributes?file=/README.md)

refers to neuefische/web-curriculum-new-format#190

### CODE REVIEW

File: sessions/react-basics/attributes/src/styles.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+}
+
+.article {
+  width: 50%;
+  margin: 2rem auto;
+  padding: 1rem 3rem;
+  background-color: var(--foam);
+  box-shadow: var(--granite) 0px 2px 8px;
+  text-align: center;
```
Comment (CJAkkuec) [inline sessions/react-basics/attributes/src/styles.css:28]:
```suggestion
```
I'm going to establish the WDACT foundation.
**W**eb **D**evelopers **A**gainst **C**entered **T**ext. 😂

File: sessions/react-basics/attributes/src/styles.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+.article__link {
+  padding: 0.5rem;
+  border-radius: 10px;
+}
+
+.article__link:hover {
+  background-color: var(--granite);
+  color: var(--water);
+}
```
Comment (CJAkkuec) [inline sessions/react-basics/attributes/src/styles.css:44]:
```suggestion
.article__link {
  width: fit-content;
  background-color: var(--granite);
  color: var(--foam);
  text-decoration: none;
  padding: 0.5rem;
  border-radius: 10px;
}

.article__link:hover {
  background-color: var(--nemo);
  color: var(--water);
}
```

Just a little suggestion 💅 ✨

**Review by CJAkkuec — CHANGES_REQUESTED** [review]
LGTM 👩‍🎤
Just some styling suggestions, and my [solution](https://codesandbox.io/s/strange-hooks-h28l6v?file=/src/App.js).

File: sessions/react-basics/attributes/src/styles.css (hunk 1 above)
Comment (PReichetanz) [reply sessions/react-basics/attributes/src/styles.css:28]: WDACT sounds good to me! I think `text-align: center` is a relic from before I have implemented `flexbox` and forgot to delete it.

**Pull request merged**